libs/utils/src/ballotPackage.ts

```diff
@@ -128,10 +128,11 @@ async function readTextEntry(zipfile: ZipFile, entry: Entry): Promise<string> {
 }
 
 async function readJSONEntry<T>(zipfile: ZipFile, entry: Entry): Promise<T> {
-  return JSON.parse(await readTextEntry(zipfile, entry))
+  const parsed = JSON.parse(await readTextEntry(zipfile, entry))
+  return parsed
```

Comment:
does this change do anything material?

Reply:
oh my bad, I did this so I could log it temporarily to debug something will switch back